### PR TITLE
Create PostgreSQL connection implmenation

### PR DESCRIPTION
**Pull request by Kidus-berhanu**
i wrote this code to connect to a database, it need to be connected to a PostgreSQL, it frist  reads data into a Pandas DataFrame, splits the data into training and testing sets, trains a logistic regression model, evaluates the model on the test data, and closes the database connection. You can modify the code as needed to suit your specific requirements and the database you are using.This code separates the logic into individual functions and encapsulates the logic of each step in the machine learning workflow. The if __name__ == '__main__': block ties everything together and runs the code as a standalone script. This makes it easier to understand and maintain the code, and allows you to reuse the functions in other scripts if necessary.

## Description

Please include a summary of the change and the issue it solves. 

**Fixes** #(issue)

## Type of change

(Please delete options that are not relevant)

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📄 This change requires a documentation update

### What is the solution?

(Describe at a high level how the feature was implemented)

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation, or created issues to update them.
- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] I have shared a short loom video or screenshots demonstrating any new functionality.

### CODE REVIEW

**Comment by Kidus-berhanu** [comment]
This code implements a simple machine learning pipeline for training and evaluating a logistic regression model. The pipeline consists of the following steps:

Connecting to a PostgreSQL database using the psycopg2 library
Reading a table from the database into a Pandas dataframe using pd.read_sql_query
Splitting the data into training and test sets using the train_test_split function from scikit-learn
Training a logistic regression model using the training data using scikit-learn's LogisticRegression class
Evaluating the model's accuracy on the test data using the score method of the model
Closing the database connection using conn.close().
***The purpose of this code is to demonstrate how to use MindsDB in a machine learning pipeline***

**Comment by ZoranPandovski** [comment]
Please check the `help-wanted` or `first-timers-only` issues if you want to contribute. Thanks :raised_hands:
